cordova-lib/package.json

```diff
@@ -42,7 +42,7 @@
     "underscore": "1.7.0",
     "unorm": "1.3.3",
     "valid-identifier": "0.0.1",
-    "xcode": "0.6.7"
+    "xcode": "https://github.com/ogoguel/node-xcode/tarball/17926117b388d08f05ec2db1e1ad8262f67a2a8d"
```

Comment:
A pull request has been sent to node-xcode (
https://github.com/alunny/node-xcode/pull/47), but in the meantime, I was
using a tarball reference to have a working version

Actually, my pull request has just been merged into node-xcode master, but
the version not yet updated into the npm repository : what should I put
into the package.json in the meantime ?
https://github.com/alunny/node-xcode.git ?

Thanks for your pointer
Cheers
Olivier

2015-06-02 11:38 GMT+02:00 Georgi Alexandrov <notifications@github.com>:

> In cordova-lib/package.json
> <https://github.com/apache/cordova-lib/pull/219#discussion_r31506402>:
>
> > @@ -42,7 +42,7 @@
> >      "underscore": "1.7.0",
> >      "unorm": "1.3.3",
> >      "valid-identifier": "0.0.1",
> > -    "xcode": "0.6.7"
> > +    "xcode": "https://github.com/ogoguel/node-xcode/tarball/17926117b388d08f05ec2db1e1ad8262f67a2a8d"
>
> You should create PR with your changes to node-xcode
> <https://github.com/alunny/node-xcode> instead of using tarball dependency
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/cordova-lib/pull/219/files#r31506402>.
>



-- 
ıllıllı ɹǝ!ʌ!ןo ıllıllı